code/game/atoms.dm

```diff
@@ -33,6 +33,8 @@
 
 	var/datum/wires/wires = null
 
+	var/list/cooldowns = list()
```

Comment:
Maybe store it in `atom/movable` instead?
Don't think turfs need it right now.

Reply:
```
#define ON_COOLDOWN(cooldown_type) LAZYACCESS(cooldowns, cooldown_type)
#define ADD_COOLDOWN(cooldown_type, timer) LAZYSET(cooldowns, cooldown_type, addtimer(VARSET_LIST_CALLBACK(cooldowns, cooldown_type, null), timer))
```
For the future, but i dont think i want to do that big sweeping change here